spp_change_request/i18n/ckb.po

```diff
@@ -602,11 +602,6 @@ msgid ""
 "Error in validation stages. No available stage to assign to this validation."
 msgstr ""
 
-#. module: spp_change_request
```

Comment:
@gonzalesedwin1123 Will the translations for `spp_dms_extra_documents` be generated automatically, or should this section be kept and changed to match the change in data/dms.xml?

Reply:
It should be kept @kneckinator and later be updated to match the change in data/dms.xml